src/toxcore/messenger/packet/user_status.rs

```diff
@@ -0,0 +1,63 @@
+/*! UserStatus struct.
+*/
+
+use nom::le_u8;
+
+use crate::toxcore::binary_io::*;
+
+/** UserStatus is a enum that holds status of user.
+
+This packet is used to transmit sender's status to a friend.
+Every time a friend become online or my status is changed,
+this packet is sent to the friend or to all friends of mine.
+
+Serialized form:
+
+Length    | Content
+--------- | ------
+`1`       | `0x32`
+`1`       | User status(0 = online, 1 = away, 2 = busy)
+
+*/
+#[derive(Clone, Copy, Debug, Eq, PartialEq)]
+#[repr(u8)]
+/// Status of user
+pub enum UserStatus {
```

Comment:
So what about this packet? Are we leaving it as a enum?

Reply:
I will revert it today.

Reply:
done